sbd.spec

```diff
@@ -97,6 +105,17 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun sbd.service
 %systemd_postun sbd_remote.service
+%else
+/sbin/chkconfig --add sbd_helper || :
+
+%preun
+/sbin/service sbd_helper stop  &>/dev/null || :
+/sbin/service sbd_remote_helper stop  &>/dev/null || :
+if [ $1 -eq 0 ]; then
+    # Package removal, not upgrade
+    /sbin/chkconfig --del sbd_helper || :
+    /sbin/chkconfig --del sbd_remote_helper || :
+fi
```

Comment:
I'm thinking mainly of diffs -- if the if/else is under the scriptlet, the immediate context is all that's needed to evaluate a diff, but if there's one big if/else, you need several pages before and after to understand the proper context.

Yeah, trying to follow it in pacemaker is what made me think of changing it :)